content/terraform/best-practices.md

```diff
@@ -169,6 +171,8 @@ retrieved by other terraform modules, or even on the command-line using tools li
 
 We are very strict about this in "root" modules (or the top-most module), because these sensitive outputs are easily leaked in CI/CD pipelines (see [`tfmask`](https://github.com/cloudposse/tfmask) for masking secrets in output only as a last resort). We are less sensitive to this in modules that are typically nested inside of other modules.
 
+Rather than outputting a secret, you may output plain text indicating where the secret is stored, for example `RDS master password is in SSM parameter /rds/master_password`.
```

Comment:
I think this is a good tip, though thinking it would be even better if it was just returning the programmatic path to the secret, that way the output can be used to access it.

Reply:
I think you should do both. Just outputting the path without explanation could be quite confusing.